app/controllers/submit.js

```diff
@@ -699,7 +714,7 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
             let model = this.get('model');
             // Saves off current server-state basics fields, so UI can be restored in case of failure
             let currentAbstract = node.get('description');
-            let currentTags = node.get('tags').slice(0);
+            let currentTags = node.get('tags').slice(0).map(fixSpecialChar);
```

Comment:
Since this is a display concern, I might suggest fixing tag display in the template rather than the controller. You'd be using the template helper instead of the util function.